lib/src/store/directory.dart

```diff
@@ -57,7 +57,7 @@ class StoreDirectory {
   FMTCTileProvider getTileProvider([
     FMTCTileProviderSettings? settings,
     Map<String, String>? headers,
-    HttpClient? httpClient,
+    BaseClient? httpClient,
   ]) =>
```

Comment:
Hi there,
It is this way to avoid an unnecessary breaking change. You can just pass `null`.
If it turns out that there are other major breaking changes, I'll update this, but as of now, this might still become a minor release (v7.3.0), and so can't have breaking changes.
Thanks for your input, I'd appreciate any other feedback you have!